_targets.R

```diff
@@ -1,64 +1,11 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
-source("2_process/src/process_and_style.R")
-source("3_visualize/src/plot_timeseries.R")
-
-options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
+options(tidyverse.quiet = TRUE)
 
-p1_targets_list <- list(
-  tar_target(
-    site_01427207_csv,
-    download_nwis_data("1_fetch/tmp/01427207.csv"),
-    format = 'file'
-  ),
-  tar_target(
-    site_01432160_csv,
-    download_nwis_data("1_fetch/tmp/01432160.csv"),
-    format = 'file'
-  ),
-  tar_target(
-    site_01435000_csv,
-    download_nwis_data("1_fetch/tmp/01435000.csv"),
-    format = 'file'
-  ),
-  tar_target(
-    site_01436690_csv,
-    download_nwis_data("1_fetch/tmp/01436690.csv"),
-    format = 'file'
-  ),
-  tar_target(
-    site_01466500_csv,
-    download_nwis_data("1_fetch/tmp/01466500.csv"),
-    format = 'file'
-  ),
-  tar_target(
-    site_data_csv,
-    combine_sites(
-    c(site_01427207_csv, site_01432160_csv, site_01435000_csv, site_01436690_csv, site_01466500_csv),
-      file_out = "1_fetch/out/site_data.csv"),
-    format ='file'
-  ),
-  tar_target(
-    site_info,
-    nwis_site_info(site_data_csv),
-  )
-)
-
-p2_targets_list <- list(
-  tar_target(
-    site_data_munged, 
-    process_data(site_data_csv, site_info)
-  )
-)
+source("1_fetch.R")
```

Comment:
Should these all be in their phase folders or in the base project working directory?

Reply:
These are in the correct spot! All of these makefiles should be top-level 👍